### ISSUE TITLE
updates for Microsoft's 20141123 build, zips, etc

### ISSUE THREAD

**Issue by jokeyrhyme** [xdissent/ievms#247]
Thought I'd mention this:
- http://blogs.msdn.com/b/ie/archive/2015/01/06/ie-test-vms-on-modern-ie-get-a-refresh.aspx

Apparently they've swapped from RAR to ZIP, and the provided images ship with IE updates blocked by default (and all other components updated as of November 2014).

**Comment by xdissent** [comment]
Oh cool, thanks. I'll take a look! Could be great news

**Comment by grayghostvisuals** [comment]
Holy :hankey: !

**Comment by xdissent** [comment]
Hmm, I'll leave this open for now, but I'm leaning towards sticking to the old since we know they work. I believe I had issues with guest additions and logins with the new images. Plus most people already have local copies of the old ones.

**Comment by jokeyrhyme** [comment]
There are also new Edge VMs, too.

**Comment by xdissent** [comment]
Yeah, we're in fact using the new Edge VM now: https://github.com/xdissent/ievms/blob/master/ievms.sh#L390
